src/test/java/org/apache/commons/lang3/time/StopWatchTest.java

```diff
@@ -500,6 +501,117 @@ void testToStringWithMessage() throws InterruptedException {
         assertEquals(SPLIT_CLOCK_STR_LEN + MESSAGE.length() + 1, splitStr.length(), "Formatted split string not the correct length");
     }
 
+    @Test
+    void testSplitsWithStringLabels() {
+        final StopWatch watch = new StopWatch();
+        final String firstLabel = "one";
+        final String secondLabel = "two";
+        final String thirdLabel = "three";
+        final int firstSleepTime = 400;
+        final int secondSleepTime = 300;
+        final int thirdSleepTime = 200;
+        watch.start();
+        ThreadUtils.sleepQuietly(Duration.ofMillis(100));
+
+        watch.split(firstLabel);
+        ThreadUtils.sleepQuietly(Duration.ofMillis(firstSleepTime));
+        watch.split(secondLabel);
+        ThreadUtils.sleepQuietly(Duration.ofMillis(secondSleepTime));
+        watch.split(thirdLabel);
+        ThreadUtils.sleepQuietly(Duration.ofMillis(thirdSleepTime));
+
+        final List<StopWatch.Split> splits = watch.getProcessedSplits();
+
+        // check sizes
+        assertEquals(3, splits.size());
+
+        // check labels
+        assertEquals(splits.get(0).getLabel(), firstLabel);
+        assertEquals(splits.get(1).getLabel(), secondLabel);
+        assertEquals(splits.get(2).getLabel(), thirdLabel);
+
+        // check duration
+        final int margin = 200;
+        assertEquals(firstSleepTime, splits.get(0).getDuration(), margin);
+        assertEquals(secondSleepTime, splits.get(1).getDuration(), margin);
+        assertEquals(thirdSleepTime, splits.get(2).getDuration(), margin);
+
+        // check report
+        final String report = watch.getReport();
+        assertTrue(report.contains("one 00:00:00."));
+        assertTrue(report.contains("two 00:00:00."));
+        assertTrue(report.contains("three 00:00:00."));
+    }
+
+    @Test
+    void testSplitsWithIntLabels() {
+        final StopWatch watch = new StopWatch();
+        final int firstLabel = 1;
+        final int secondLabel = 2;
+        final int thirdLabel = 3;
+        final int firstSleepTime = 400;
+        final int secondSleepTime = 300;
+        final int thirdSleepTime = 200;
+        watch.start();
+        ThreadUtils.sleepQuietly(Duration.ofMillis(100));
+
+        watch.split(firstLabel);
+        ThreadUtils.sleepQuietly(Duration.ofMillis(firstSleepTime));
+        watch.split(secondLabel);
+        ThreadUtils.sleepQuietly(Duration.ofMillis(secondSleepTime));
+        watch.split(thirdLabel);
+        ThreadUtils.sleepQuietly(Duration.ofMillis(thirdSleepTime));
+
+        final List<StopWatch.Split> splits = watch.getProcessedSplits();
+
```

Comment:
Remove all these blank lines. If you want to call something out, use an `// inline comment` like you already do here.